snuba/querylog/query_metadata.py

```diff
@@ -92,13 +92,15 @@ class SnubaQueryMetadata:
     dataset: str
     timer: Timer
     query_list: MutableSequence[ClickhouseQueryMetadata]
+    anonymized_request_body: str
 
     def to_dict(self) -> Dict[str, Any]:
         return {
             "request": {
                 "id": self.request.id,
                 "body": self.request.body,
                 "referrer": self.request.referrer,
+                "anonymized_request_body": self.anonymized_request_body,
```

Comment:
This is not part of the request object. 
I would  move it outside the `request` dictionary and just have it right before dataset.
Also would you mind calling it `snql_anonymized`. That's what the consumer expects